packages/server/src/modules/utils/tests/shuffleArray.spec.ts

```diff
@@ -0,0 +1,14 @@
+import application from '@application';
+
+beforeAll(() => application.test.start());
+
+describe('Shuffle array', () => {
```

Comment:
Remove shuffle from utils, just use lodash. Not needed as an util.

Reply:
You gotta reload before re-reviewing 😛
![Screenshot 2023-09-19 at 11 53 26](https://github.com/MrBr/chess-tent/assets/34143527/0be2ad4e-ead5-4e8a-b9ef-fdd31586b157)